--- origsrc/wv-1.2.3/GNUmakefile.am	2005-07-24 11:23:05.000000000 -0500
+++ src/wv-1.2.3/GNUmakefile.am	2006-10-19 21:23:02.179375000 -0500
@@ -31,7 +31,7 @@
 wvVersion_LDADD = libwv.la $(WVLIBS) $(GETOPT)
 
 libwv_la_LIBADD =  $(WVLIBS)
-libwv_la_LDFLAGS = -version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) -release $(LT_RELEASE) -export-dynamic
+libwv_la_LDFLAGS = -version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) -release $(LT_RELEASE) -export-dynamic -no-undefined
 libwv_la_SOURCES = \
 	winmmap.c \
 	list.c \
