diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x Makefile.in.in -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x GNUmakefile.in -x configure -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x omf.make -x xmldocs.make origsrc/wv-1.2.3/GNUmakefile.am src/wv-1.2.3/GNUmakefile.am
--- origsrc/wv-1.2.3/GNUmakefile.am	2005-07-24 11:23:05.000000000 -0500
+++ src/wv-1.2.3/GNUmakefile.am	2006-10-19 21:23:02.179375000 -0500
@@ -14,7 +14,7 @@
 
 wvhdrdir = $(includedir)/wv
 
-wvhdr_HEADERS = wv.h
+wvhdr_HEADERS = wv.h wvinternal.h
 
 bin_SCRIPTS = wvCleanLatex wvHtml wvDocBook wvAbw wvMime wvPS wvDVI wvLatex wvPDF wvText wvWml
 
@@ -31,7 +31,7 @@
 wvVersion_LDADD = libwv.la $(WVLIBS) $(GETOPT)
 
 libwv_la_LIBADD =  $(WVLIBS)
-libwv_la_LDFLAGS = -version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) -release $(LT_RELEASE) -export-dynamic
+libwv_la_LDFLAGS = -version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) -release $(LT_RELEASE) -export-dynamic -no-undefined
 libwv_la_SOURCES = \
 	winmmap.c \
 	list.c \
@@ -135,8 +135,7 @@
 	rc4.h \
 	roman.h \
 	utf.h \
-	winmmap.h \
-	wvinternal.h
+	winmmap.h
 
 OTHERSCRIPTS = wvMime
 
